jaggery-modules/utils/module/scripts/response/response.js

```diff
@@ -93,4 +93,35 @@ var log = new Log("response");
          return resp;
      };
 
+    response.buildErrorResponseDefault = function (statusCode, desc, resp, msg, moreInfo, error) {
+        var errObj = {};
+        errObj.code = statusCode;
```

Comment:
If the JSON was constructed using literal notation, it would be more cleaner. i.e.

var errObj = {
    code: statusCode,
    error: error,
    description: description,
    message: msg,
    moreInfomation: moreInfo
};